pkg/volume/gce_pd/attacher.go

```diff
@@ -165,6 +165,9 @@ func (attacher *gcePersistentDiskAttacher) MountDevice(spec *volume.Spec, device
 	options := []string{}
 	if readOnly {
 		options = append(options, "ro")
+	} else {
+		// as per https://cloud.google.com/compute/docs/disks/add-persistent-disk#formatting
+		options = append(options, "discard")
```

Comment:
OK I looked at it even deeper. enabling this in mkfs only speeds up mkfs, it has no fs impact once mounted. We get benefits of "discard" in the mount option. So we don't need to do both.